### ISSUE TITLE
Create a CONTRIBUTING.md

### ISSUE THREAD

**Issue by Jimbo4350** [IntersectMBO/cardano-node#3794]
Relevant: https://input-output-rnd.slack.com/archives/C48HA39RS/p1649781655400349

**Comment by jonathangenlambda** [comment]
Yes please! It is utterly difficult and confusiong to get started with PRs if it is undefined / unclear which branch to use, what rules there are and so on...

**Comment by Jimbo4350** [comment]
Update the `Contributing.rst` to indicate how to actually contribute!
